spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -98,26 +55,31 @@ def create_workspace_csv(workspaces, client_id_column)
     @config[:ads_client][:jdbc_url] = @ads.data['connectionUrl']
 
     @ads_client = GoodData::Datawarehouse.new(
-        @config[:ads_client][:username],
-        @config[:ads_client][:password],
-        @config[:ads_client][:ads_id],
-        jdbc_url: @ads.data['connectionUrl']
+      @config[:ads_client][:username],
+      @config[:ads_client][:password],
+      @config[:ads_client][:ads_id],
+      jdbc_url: @ads.data['connectionUrl']
     )
 
     @suffix = LcmHelper.create_suffix
     @release_table_name = "LCM_RELEASE_#{@suffix}"
     LcmHelper.create_release_table(@release_table_name, @ads_client)
```

Comment:
warning: Similar code found in shared_contexts_for_lcm.rb:44, shared_contexts_for_lcm.rb:91 (mass = 32) (flay)